packages/netlify-cms-widget-markdown/src/MarkdownControl/RawEditor.js

```diff
@@ -107,6 +107,7 @@ export default class RawEditor extends React.Component {
             buttons={field.get('buttons')}
             disabled
             rawMode
+            onlyRawMode={field.get('onlyRawMode')}
```

Comment:
Thanks @pchorus! Initially I thought we just need to change this to snake case (e.g. `only_raw_mode`), but how about having:
`mode: ['rich_text', 'raw']`

We would also need to add this to the docs and the widget [schema](https://github.com/netlify/netlify-cms/blob/8d693caebdda381b989ea9b934fac4e65d06498e/packages/netlify-cms-widget-markdown/src/schema.js)

Reply:
Thanks for your feedback, @erezrokah. I also thought about an array option.
I made the changes you proposed: I added the option `mode: ['rich_text', 'raw']` and added it to the schema and the docs.
I hope it meets your expectations.